explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -505,42 +410,20 @@ private object SpectroscopyModesTable extends ModesTableCommon:
             InstrumentConfigAndItcResult(row.entry.instrument, row.result.toOption)
 
         def scrollButton(content: VdomNode, style: Css, indexCondition: Int => Boolean): TagMod =
-          selectedIndex.value.whenDefined(
-            using
-            idx =>
-              Button(
-                clazz = ExploreStyles.ScrollButton |+| style,
-                severity = Button.Severity.Secondary,
-                onClick = scrollToVirtualizedIndex(idx, virtualizerRef)
-              ).withMods(content).compact.when(indexCondition(idx))
+          selectedIndex.value.whenDefined(idx =>
+            Button(
+              clazz = ExploreStyles.ScrollButton |+| style,
+              severity = Button.Severity.Secondary,
+              onClick = scrollToVirtualizedIndex(idx, virtualizerRef)
+            ).withMods(content).compact.when(indexCondition(idx))
           )
 
-        def renderName(name: Option[NonEmptyString]): String =
-          name.fold("")(n => s"$n: ")
-
-        val errLabel: List[VdomNode] =
```

Comment:
I extracted and retained the `errLabel` code, but I wasn't able to get an error in the list. I think for most of the errors this checks for we don't actually call the ITC?